chia/cmds/wallet_funcs.py

```diff
@@ -141,6 +141,10 @@ async def get_wallet(wallet_client: WalletRpcClient, fingerprint: int = None) ->
         return None
     if len(fingerprints) == 1:
         fingerprint = fingerprints[0]
+    if fingerprint is None:
+        config = load_config(DEFAULT_ROOT_PATH, "config.yaml", SERVICE_NAME)
+        if 'default_fingerprint' in config:
```

Comment:
Instead of `if` you can go with `fingerprint = config.get("default_fingerprint")`
on `L148` we check for `is not None` so thats ok